src/PIL/Image.py

```diff
@@ -3000,7 +3000,7 @@ def frombuffer(mode, size, data, decoder_name="raw", *args):
         if args == ():
             args = mode, 0, 1
         if args[0] in _MAPMODES:
-            im = new(mode, (1, 1))
+            im = new(mode, (0, 0))
```

Comment:
For the record. This call gives 1.37 µs instead of 1.47 µs, which probably unmeasurable for whole `frombuffer` function.